extensions/vscode-api-tests/package.json

```diff
@@ -41,13 +41,12 @@
 		}
 	},
 	"scripts": {
-		"compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
-		"vscode:prepublish": "node ../../node_modules/gulp/bin/gulp.js --gulpfile ../../build/gulpfile.extensions.js compile-extension:vscode-api-tests ./tsconfig.json"
+		"compile": "gulp compile-extension:vscode-api-tests",
+		"watch": "gulp watch-extension:vscode-api-tests"
```

Comment:
The extension shouldn't have an implicit dependency on the *parent* project/package.json. Now it misses the dependency to `gulp` and `typescript`. I'd reckon to simply update the `typescript` dependency here